UserScrape/app.py

```diff
@@ -16,13 +20,19 @@
     user_seed_videos = seeds_for_user(user, video_seeds_df)
 
     try:
-        crawler.load_home_and_login()
-        # crawler.login()
-        # for video in user_seed_videos:
-        #     crawler.get_recommendations_for_video(video.video_id)
+        # crawler.load_home_and_login()
+        crawler.login()
+        crawler.delete_history()
+        for repetition in range(1,repetitions):
+            asyncio.run(crawler.watch_videos([video.video_id for video in user_seed_videos[0:5]]))
```

Comment:
I restricted this list to the first 5 just for testing the loop once, for the final pull request this would be ofc the full list